src/conky.h

```diff
@@ -298,6 +298,12 @@ void generate_text_internal(char *, int, struct text_object);
 int percent_print(char *, int, unsigned);
 void human_readable(long long, char *, int);
 
+#ifdef BUILD_X11
```

Comment:
Variable `utf8_mode` is defined inside of `#ifdef BUILD_X11 ... #endif` in `conky.cc`, so if it will be used when `BUILD_X11` is not defined then the program won't link.